wl/server/MuninStatistics.java

```diff
@@ -52,43 +65,58 @@ private MuninStatistics() {
 		commandCounters = new long[Command.values().length];
 		for (int i = 0; i < commandCounters.length; i++) commandCounters[i] = 0;
 
-		currentUnregisteredUsers = 0;
+		registeredUsers = 0;
+		unregisteredUsers = 0;
 		failedLogins = 0;
-		successfulLogins = 0;
 		skippedCommands = 0;
 		successfulCommands = 0;
-		currentRegisteredUsers = new ArrayList<>();
+		clientLifetimes = new ArrayList<>();
 		allRegisteredUsers = new HashSet<>();
 		cmdInfoToSkip = new HashMap<>();
 	}
 
+	/**
+	 * Print all current statistics.
+	 * @param version Munin protocol version.
+	 * @param out The stream to print data to.
+	 * @throws Exception If anything at all goes wrong, throw an %Exception.
+	 */
 	public synchronized void printStats(int version, PrintStream out) throws Exception {
-		switch (version) {
-			case 1: {
-				out.println(System.currentTimeMillis() - initTime);
-				out.println(currentRegisteredUsers.size());
-				out.println(currentUnregisteredUsers);
-				out.println(allRegisteredUsers.size());
-				out.println(successfulLogins);
-				out.println(failedLogins);
-
-				long totalCmd = skippedCommands;
-				for (long cmd : commandCounters) {
-					totalCmd += cmd;
-					out.println(cmd);
-				}
-				out.println(totalCmd - successfulCommands);
+		out.println((System.currentTimeMillis() - initTime) / (1000.0 * 60 * 24));
```

Comment:
Using only ints here would cause the division to be floored, but I want double values for a continuous graph. Using a floating-point constant in a multiplication is more readable than explicitly casting something to float or double, IMHO.